recipes/list_items/contact_info/contact_info.stories.js

```diff
@@ -287,8 +302,16 @@ Variants.parameters = {
       <template #subtitle>+1 (415) 123-4567</template>
       <template #right>
         <div class="d-d-flex d-ai-center d-m16">
-          <icon-chat class="d-m4" />
-          <icon-menu-horizontal class="d-m4" />
+          <dt-button importance="clear" aria-label="chat">
```

Comment:
For both icons, change props so it winds up as `d-btn d-btn--xs d-btn--circle`. Appearance should be like the attached video after doing this and the other adjustments commented below.

https://user-images.githubusercontent.com/1165933/202616332-6d2d18c2-a249-48de-b757-c7c817d1daf3.mov